Assets/Project/Scripts/Modules/StartUpScene/StartUpDirector.cs

```diff
@@ -14,15 +11,15 @@ public class StartUpDirector : MonoBehaviour
     {
         [SerializeField] private GameObject _startButton;
 
-        private IEnumerator Start()
+        private async void Start()
         {
             // Don't destroy EventSystem
             var eventSystem = FindObjectOfType<UnityEngine.EventSystems.EventSystem>();
             if (eventSystem != null)
                 DontDestroyOnLoad(eventSystem.gameObject);
 
             // UIManager Initialize
-            yield return new WaitWhile(() => !UIManager.Instance.Initialized);
+            await UniTask.WaitUntil(() => UIManager.Instance.Initialized);
```

Comment:
以下のようにして、 `UIManager` の `Initialized` を消しましょう
```suggestion
            await UIManager.Instance.AwakeAsync();
```

Reply:
b8215b8

Reply:
`UIManager.Instance.AwakeAsync();` を使わないのには理由がある？

Reply:
AwakeAsyncって何か特別な意味ある？

Reply:
ある。特定の `gameObject` の `Awake` を待ってくれる。[公式ドキュメント](https://cysharp.github.io/UniTask/api/Cysharp.Threading.Tasks.Triggers.AsyncAwakeTrigger.html#Cysharp_Threading_Tasks_Triggers_AsyncAwakeTrigger_AwakeAsync)

Reply:
まさに今回使いたい api

Reply:
なるほど、勉強不足でした